app/transport/http/resp.go

```diff
@@ -18,16 +20,16 @@ type Response struct {
 //  after an error occurs
 
 // WriteJSONResponse writes JSON response
-func WriteJSONResponse(rw http.ResponseWriter, statusCode int, data interface{}) {
-	rw.Header().Set("Content-Type", "application/json")
-	rw.WriteHeader(statusCode)
+func WriteJSONResponse(w http.ResponseWriter, statusCode int, data any, logger *lgr.Logger) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(statusCode)
 
 	if data == nil {
-		log.Println("Data is empty") // we need to log the error when we select the logger
+		logger.Warnf("%s\n", "Data is empty")
```

Comment:
I'm not sure if we want to log this. Please recall a case with 204 code. 
It's a valid case, why should we log your message in such a case as a worning?

Reply:
Sure. You are right, we do not need to catch status code and will leave only naked return.